pkg/services/publicdashboards/service/query.go

```diff
@@ -394,11 +394,16 @@ func buildTimeSettings(d *dashboards.Dashboard, reqDTO models.PublicDashboardQue
 }
 
 // returns from, to and timezone from the request if the timeSelection is enabled or the dashboard default values
-func getTimeRangeValuesOrDefault(reqDTO models.PublicDashboardQueryDTO, d *dashboards.Dashboard, timeSelectionEnabled bool) (string, string, *time.Location) {
+func getTimeRangeValuesOrDefault(reqDTO models.PublicDashboardQueryDTO, d *dashboards.Dashboard, timeSelectionEnabled bool, panelID int64) (string, string, *time.Location) {
 	from := d.Data.GetPath("time", "from").MustString()
 	to := d.Data.GetPath("time", "to").MustString()
 	dashboardTimezone := d.Data.GetPath("timezone").MustString()
 
+	panelRelativeTime := getPanelRelativeTimeRange(d.Data, panelID)
+	if panelRelativeTime != "" {
+		from = panelRelativeTime
```

Comment:
as it's a relative time range, there's no need for the `to` param, correct?

Reply:
Yes exactly!